Dockerfile

```
@@ -0,0 +1,32 @@
+FROM ruby:3.0.0-alpine as Builder
+
+RUN apk add --update --no-cache \
+    build-base \
+    postgresql-dev \
+    git \
+    tzdata
+
+RUN gem install bundler
+
+WORKDIR /simple_analytics
+
+COPY . /simple_analytics/
+
+RUN bundle install
+
+FROM ruby:3.0.0-alpine as Final
+
+RUN apk add --update --no-cache \
+    build-base \
+    postgresql-client \
```

On mac without gcompat package docker is not work

```suggestion
RUN apk add --update --no-cache \
    build-base \
    gcompat \
    postgresql-client \
```